client/source/class/cv/io/transport/Sse.js

```diff
@@ -132,18 +152,22 @@ qx.Class.define('cv.io.transport.Sse', {
       }
     },
 
-    __addRecordedEventListener: function(topic) {
-      this.debug('subscribing to topic '+topic);
-      this.eventSource.addEventListener(topic, function(e) {
-        this.dispatchTopicMessage(topic, e);
-      }.bind(this), false);
+    __addRecordedEventListener(topic) {
+      this.debug('subscribing to topic ' + topic);
+      this.eventSource.addEventListener(
+        topic,
+        function (e) {
+          this.dispatchTopicMessage(topic, e);
+        }.bind(this),
+        false
+      );
```

Comment:
Another example where I think the older, compatcer style was more easy to read and understand

Reply:
This case is indeed an opinion that prettier and I don't share. And also it is not possible to avoid this change.